libs/core/src/lib/table/directives/table-cell.directive.ts

```diff
@@ -73,12 +77,17 @@ export class TableCellDirective implements AfterContentInit {
 
     /** @hidden */
     ngAfterContentInit(): void {
+        const elementRef = this.elementRef.nativeElement;
+        if (elementRef.title && elementRef.textContent && this.ariaLabel === null) {
+            this.ariaLabel = elementRef.title + elementRef.textContent; 
```

Comment:
It may be not clear for the customer why is the `aria-label` set when a title is set. Let's remove it.